package.json

```diff
@@ -20,13 +20,7 @@
   "workspaces": [
     "alchemy",
     "alchemy-web",
-    "examples/aws-app",
-    "examples/cloudflare-nuxt-pipeline",
-    "examples/cloudflare-react-router",
-    "examples/cloudflare-redwood",
-    "examples/cloudflare-vite",
-    "examples/cloudflare-worker-bootstrap",
-    "examples/cloudflare-worker"
+    "examples/*"
   ],
```

Comment:
I should omit tanstack from this since it doesn't work in monorepo right now, or i should mayeb try pnpm instead